### ISSUE TITLE
options "auditing_services", "storage_services" do not exist

### ISSUE THREAD

**Issue by frostieDE** [DamienHarper/auditor-bundle#334]
| Q                           | A
| ----------------------------| -----------------------
| `auditor-bundle` version    | 5.2.0
| `auditor` version | 2.3.1
| PHP version                 | 8.1.7
| Database                    | MySQL 

#### Summary

After a recent update (using `composer update`) I receive an error. I suppose it has something to do with changes in [this](https://github.com/DamienHarper/auditor/pull/135) commit as some options are removed (which may be configured by this bundle?)

My configuration is quite simple:

```yaml
# Full configuration reference available at:
# https://damienharper.github.io/auditor-docs/docs/auditor-bundle/configuration/reference.html
dh_auditor:
    enabled: '%env(bool:AUDIT_ENABLED)%'
    timezone: 'Europe/Berlin'
    providers:
        doctrine:
            viewer: true
            table_prefix: null
            table_suffix: '_audit'

```

When reverting to `auditor` version `2.3.0` this error is gone.

#### Current behavior

When running `php bin/console cache:clear`:

```
In OptionsResolver.php line 871:
                                                                                                                                                                                     
  The options "auditing_services", "storage_services" do not exist. Defined options are: "entities", "ignored_columns", "storage_mapper", "table_prefix", "table_suffix", "viewer".  
                                                                                                                                                                                     

```

#### How to reproduce

Update to recent version of both `auditor` and `auditor-bundle`.

#### Expected behavior

No error :)

**Comment by DamienHarper** [comment]
@frostieDE Well, this is a regression from latest auditor version. Will fix it asap, sorry for the inconvenience.

**Comment by DamienHarper** [comment]
@frostieDE  I released `2.3.2` which should fixes the issue. Feel free to reopen if needed.

**Comment by frostieDE** [comment]
> Will fix it asap, sorry for the inconvenience.

Thanks for the quick fix.
